src/lv_api_map_v8.h

```diff
@@ -249,6 +249,64 @@ static inline void lv_obj_move_background(lv_obj_t * obj)
 #define lv_image_decoder_built_in_open      lv_bin_decoder_open
 #define lv_image_decoder_built_in_close     lv_bin_decoder_close
 
+/**
+ * Deprecated API, use `lv_draw_buf_destroy` instead.
+ *
+ * Free the snapshot image returned by @ref lv_snapshot_take
+ * @param dsc   the image descriptor generated by lv_snapshot_take.
+ */
+static inline void lv_snapshot_free(lv_image_dsc_t * dsc)
+{
+    LV_LOG_WARN("Deprecated API, use lv_draw_buf_destroy instead.");
+    lv_draw_buf_destroy((lv_draw_buf_t *)dsc);
+}
+
+/**
+ * Deprecated API, use lv_snapshot_take_to_draw_buf instead.
+ * Take snapshot for object with its children, save image info to provided buffer.
+ * @param obj       the object to generate snapshot.
+ * @param cf        color format for generated image.
+ * @param dsc       image descriptor to store the image result.
+ * @param buf       the buffer to store image data. It must meet align requirement.
+ * @param buf_size  provided buffer size in bytes.
+ * @return          LV_RESULT_OK on success, LV_RESULT_INVALID on error.
+ */
+static inline lv_result_t lv_snapshot_take_to_buf(lv_obj_t * obj, lv_color_format_t cf, lv_image_dsc_t * dsc,
+                                                  void * buf,
+                                                  uint32_t buf_size)
+{
+    lv_draw_buf_t draw_buf;
+    LV_LOG_WARN("Deprecated API, use lv_snapshot_take_to_draw_buf instead.");
+    lv_draw_buf_init(&draw_buf, 1, 1, cf, buf_size, buf, buf_size);
+    lv_result_t res = lv_snapshot_take_to_draw_buf(obj, cf, &draw_buf);
```

Comment:
Note these warnings.

Reply:
I have removed the warning logs and added lv_image_dsc.h

Reply:
The warnings are still here.

Reply:
Ah, get it. Now should be fixed.